lib/token_mods.py

```diff
@@ -55,9 +56,12 @@ def use_token_copy(tokenfile: str) -> str:
 def get_token_scope(tokenfilename: str) -> List[str]:
     """get the list of scopes from our token file"""
 
-    with os.popen(f"decode_token.sh -e scope {tokenfilename}", "r") as sf:
-        data = sf.read()
-        scopelist = data.strip().strip('"').split(" ")
+    with open(tokenfilename) as f:
```

Comment:
I think this is good for now, but in the future, time permitting, we should look at refactoring the 

```python
with open(tokenfilename) as f:
    ...
    token = dict(....)
```

bit as a single function that can be shared.  I'm not sure where the correct place for such a function would be though...

Reply:
This function could even handle the raising of the right scitokens `Exception` in the case where it's needed.